src/extensions/react-flow/nodes/NodeContent.tsx

```diff
@@ -210,6 +212,10 @@ export interface NodeContentProps<NODE_DATA, NODE_CONTENT_PROPS = any>
      * width and height dimensions of the node (Optional)
      */
     nodeDimensions?: NodeDimensions;
+    /** if node is resizable, this allows direction of specificity */
+    resizeDirections?: Enable;
```

Comment:
As we do not support all resize directions, only the supported directions from `Enable` should be available via `resizeDirections`.